src/test/java/uk/gov/hmcts/ccd/endpoint/ui/QueryEndpointTest.java

```diff
@@ -146,6 +146,13 @@ void shouldThrowBadRequest() {
         assertThrows(BadRequestException.class, () -> queryEndpoint.getJurisdictions("creat"));
     }
 
+    @Test
+    @DisplayName("Should throw bad request Exception for invalid case type id")
+    void shouldThrowBadRequestForInvalidCaseType() {
+        assertThrows(BadRequestException.class, () -> queryEndpoint.findWorkbasketInputDetails("22",
```

Comment:
Please add plenty more assertions of this type for more variety of bad case type id strings to be covered at unit test level.